client/model.go

```diff
@@ -53,17 +53,13 @@ type EnvironmentCreate struct {
 }
 
 type DeployRequest struct {
-	BlueprintId          string                  `json:"blueprintId,omitempty"`
-	BlueprintRevision    string                  `json:"blueprintRevision,omitempty"`
-	BlueprintRepository  string                  `json:"blueprintRepository,omitempty"`
-	ConfigurationChanges *[]ConfigurationChanges `json:"configurationChanges,omitempty"`
-	TTL                  *TTL                    `json:"ttl,omitempty"`
-	EnvName              string                  `json:"envName,omitempty"`
-	UserRequiresApproval bool                    `json:"userRequiresApproval,omitempty"`
-	Targets              string                  `json:"targets,omitempty"`
-	// TODO: not sure about this one
-	CustomEnv0EnvironmentVariables *interface{} `json:"customEnv0EnvironmentVariables,omitempty"`
-	TriggerName                    string       `json:"triggerName,omitempty"`
```

Comment:
Please notice that I've removed targets, customEnv0EnvVars, triggerName as I understand that only we send those (even though nothing stops people from sending them via rest api)
Should I maybe send go provider as a trigger? (honestly this can be addressed later, just wanted to point it out)